### ISSUE TITLE
overwrite not recognized on pcs cluster setup

### ISSUE THREAD

**Issue by sean-freeman** [linux-system-roles/ha_cluster#124]
Ansible Task 'Create a corosync.conf file content using pcs-0.10' has an error on re-run using RHEL 8.4 with `pcs cluster setup`

### Error:
`option --overwrite not recognized`

### Diagnosis

This option does not exist, is this supposed to be `--force` ?

Version:
```
[root@host-p ~]# pcs --version
0.10.4
```

### Output:
```json
TASK [fedora.linux_system_roles.ha_cluster : Create a corosync.conf file content using pcs-0.10] *********
fatal: [host-p]: FAILED! =>
{
    "changed": true,
    "cmd": [
        "pcs",
        "cluster",
        "setup",
        "--corosync_conf",
        "/tmp/ansible.5vjb9txg_ha_cluster_corosync_conf",
        "--overwrite",
        "--",
        "clusterhdb",
        "host-p",
        "host-s"
    ],
    "delta": "0:00:00.299788",
    "end": "2023-04-27 23:38:30.909534",
    "msg": "non-zero return code",
    "rc": 1,
    "start": "2023-04-27 23:38:30.609746",
    "stderr": "",
    "stderr_lines": [],
    "stdout_lines": [
        "option --overwrite not recognized"
    ]
}
```

**Comment by richm** [comment]
I'll have to defer to @tomjelinek - maybe the option was added/changed after rhel 8.4?

**Comment by tomjelinek** [comment]
Hi @sean-freeman,

The `--overwrite` option was introduced in pcs-0.10.6 which was released in RHEL 8.3. The option was added specifically for the role to be able to setup clusters and that's why the role requires RHEL 8.3 or later. I'm not sure why you have pcs-0.10.4 on a RHEL 8.4 machine. RHEL 8.4 ships pcs-0.10.8.

**Comment by sean-freeman** [comment]
@tomjelinek I apologise profusely, I immediately double checked and can now see my Ansible Task to upgrade my OS Image from RHEL 8.2 to RHEL 8.4 was being skipped -- an accidental change during testing for the PR I raised.

Closing this as end-user error.
